docs/en/02_Developer_Guides/19_GraphQL/01_getting_started/02_configuring_your_schema.md

```diff
@@ -94,11 +98,11 @@ SilverStripe\GraphQL\Schema\Schema:
       src:
         - app/_graphql
         - module/_graphql
+        # The next line would map to `vendor/somevendor/somemodule/_graphql`
+        - 'somevendor/somemodule: _graphql'
```

Comment:
I've updated this example from my initial change:
- we need to quotes, otherwise this gets confused for an array
- While this uses the same syntax as the vendor expose logic, the folder itself doesn't need to be expose for it to work.